src/main/java/pl/joegreen/lambdaFromString/LambdaFactoryConfiguration.java

```diff
@@ -19,6 +19,7 @@
     private String compilationClassPath;
     private ClassLoader parentClassLoader;
     private JavaCompiler javaCompiler;
+    private boolean enablePreview;
```

Comment:
Is this required in some circumstances for compilation on java 14 or is it just a separate feature you've decided to add to support this additional compiler parameter?

Reply:
This is a separate feature we added to be able to use [preview features of Java](https://openjdk.java.net/jeps/12) from within `lambdaFromString`.